### DISCUSSION TITLE
Cannot add subdomain to my Vercel test project

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#1606] [Help] [answered]
### Summary

Hi all,

I'm trying to demo a problem with wildcards, subdomains and middleware in nextjs. But I cannot add a wildcard subdomain:
<img width="984" alt="image" src="https://user-images.githubusercontent.com/73357914/220878618-ee9fd147-916b-435a-b6d6-18872caa504e.png">

It was working fine with another project:
<img width="979" alt="image" src="https://user-images.githubusercontent.com/73357914/220878771-58aafc60-152b-49d9-8e54-325b2d3aff86.png">

Any help on this? I'm clueless.

**Comment by amyegan** [comment, marked as answer]
HI @ddegooijer! A wildcard subdomain will only work with your username (or team name in the case of teams). Something like `*.ddegooijer.vercel.app`. You can also set up [wildcard subdomains](https://vercel.com/docs/concepts/projects/domains#subdomains,-wildcard-domains,-and-apex-domains:~:text=You%20can%20also-,configure,-wildcard%20domains.%20Using) with a [custom domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#) for a more customized address
